railties/lib/rails/generators/rails/app/templates/Dockerfile.tt

```diff
@@ -72,7 +72,9 @@ RUN bundle exec bootsnap precompile app/ lib/
 <% end -%>
 <% unless options.api? || skip_asset_pipeline? -%>
 # Precompiling assets for production without requiring secret RAILS_MASTER_KEY
-RUN SECRET_KEY_BASE_DUMMY=1 ./bin/rails assets:precompile
+ARG RAILS_MASTER_KEY
+ENV RAILS_MASTER_KEY=$RAILS_MASTER_KEY
```

Comment:
This is being set as such to not expose the MASTER_KEY in the logs when building container.

Reply:
`SECRET_KEY_BASE_DUMMY` is supposed to make so we don't need the RAILS_MASTER_KEY. Why is that not workin?